scalaz-test/src/main/scala/ztest/data.scala

```diff
@@ -0,0 +1,30 @@
+package scalaz.test
```

Comment:
Some of these files are in `ztest` directory, not `scalaz/test `; is that intentional?

Reply:
Nope.